docs/iot/guides/update-control-api.md

```diff
@@ -0,0 +1,142 @@
+
+## Checking the Latest Version
+
+The device can query the latest version of current platform to server. It can be implemented simply by:
+```cpp
+int error_code;
+
+error_code = update_control_check_new_version();
+if (error_code != UPDATE_CONTROL_ERROR_NONE)
+  return;
+```
+
+If the newer version of platform is available, the value of `UPDATE_CONTROL_PROPERTY_UPDATE_AVAILABLE` will be set properly
+(It's type and value are depend on platform developer).
+The application can get the result through the `update_control_get_property()` function:
+```cpp
+int error_code;
+bool is_available;
+
+error_code = update_control_get_property(UPDATE_CONTROL_PROPERTY_UPDATE_AVAILABLE, (void **)&is_available);
+if (error_code == UPDATE_CONTROL_ERROR_NONE) {
+  if (is_available)
+    // New version platform is available
+    ...
+}
+```
+
+## Downloading and Updating to New Version
+
+If the newer version is available, the application can download it and update to the found one.
+The functions `update_control_download_package()` and `update_control_do_update()` are offered and their usage is similar to `update_control_get_property()`:
+```cpp
+int error_code;
+
+error_code = update_control_download_package();
+if (error_code != UPDATE_CONTROL_ERROR_NONE)
+  return;
+
+error_code = update_control_do_update();
+if (error_code != UPDATE_CONTROL_ERROR_NONE)
+  return;
+```
+Usually, downloading and updating are processed sequentially but the application also can call each function separately for intending operation.
+
+## Making an Update Reservation
+
+The device especially IoT may be critical at it's operation time. Because updating procedure may include unavailable time (ex. rebooting), it needs to schedule software update at specific time.
+In this case, the application can use `update_control_make_reservation()` function:
+```cpp
+int error_code;
+struct tm time;
+
+// Set the "time" to trigger the update
+error_code = update_control_make_reservation(&time);
+if (error_code != UPDATE_CONTROL_ERROR_NONE)
+  return;
+```
+The reserved update task also can be canceled by `update_control_cancel_reservation()`.
+
+> **Note**
+>
+> It assumes only one update reservation can be set. If the platform supports multiple reservations, they should be managed properly about cancel request.
+
+## Update Properties
+
+Besides `UPDATE_CONTROL_PROPERTY_UPDATE_AVAILABLE` mentioned at above, there are several information about update.
+They can be got through `update_control_get_property()` function and each key:
+
+| Key                                   |  Description                                  | Type (example) |
+| --------------------------------------| --------------------------------------------- |----------------|
+| `UPDATE_CONTROL_PROPERTY_NEW_VERSION` | The value of new version (ex. string, number) | string         |
```

Comment:
(For example, string, and number)
Global: Please add period at the end of the sentence for description in the table

Reply:
Done.